server/src/controllers/seasonTicketTypePriceDefaultController.ts

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable camelcase */
+import {Router, Request, Response} from 'express';
+import {checkJwt, checkScopes} from '../auth';
+import {Prisma} from '@prisma/client';
+import {extendPrismaClient} from './PrismaClient/GetExtendedPrismaClient';
+import {InvalidInputError} from './eventInstanceController.service';
+
+const prisma = extendPrismaClient();
+export const seasonTicketTypePriceDefaultController = Router();
+
+// All further routes require appropriate authentication
+seasonTicketTypePriceDefaultController.use(checkJwt);
+seasonTicketTypePriceDefaultController.use(checkScopes);
+
+interface SeasonTicketTypePriceDefaultRequestItem {
+    tickettypeid_fk: number;
+    price: number;
+    concessionprice: number;
+}
+
+/**
+ * @swagger
+ * /2/season-ticket-type-price-default/{id}:
+ *   get:
+ *     summary: get season ticket type prices for a given season
```

Comment:
Might be good to say that is is by season id

Reply:
Updated